src/test/java/com/znsio/sample/e2e/screen/indigo/web/GiftVoucherScreenWeb.java

```diff
@@ -0,0 +1,175 @@
+package com.znsio.sample.e2e.screen.indigo.web;
+
+import com.znsio.e2e.tools.Driver;
+import com.znsio.e2e.tools.Visual;
+import com.znsio.sample.e2e.entities.GIFT_VOUCHER_TEST_CONTEXT;
+import com.znsio.sample.e2e.screen.indigo.GiftVoucherScreen;
+import com.znsio.sample.e2e.screen.indigo.PaymentScreen;
+import org.apache.commons.lang.NotImplementedException;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+
+import java.util.Map;
+
+public class GiftVoucherScreenWeb extends GiftVoucherScreen {
```

Comment:
You are not doing any visual validations anywhere in any of the screens. Update appropriately

Reply:
added visual validations - done